mlflow/pyspark/ml/__init__.py

```diff
@@ -6,8 +6,8 @@
 import weakref
 
 import mlflow
-from mlflow.entities import Metric, Param
 from mlflow.tracking.client import MlflowClient
```

Comment:
This one was giving some initialization issues?

Reply:
This is a mistake on my side, will revert it. not sure how tests cases were passing.